src/librustc_codegen_ssa/traits/coverageinfo.rs

```diff
@@ -0,0 +1,38 @@
+use crate::coverageinfo::CounterOp;
+//use crate::coverageinfo::map::FunctionCoverageRegions;
```

Comment:
Nit: There's some commented out code in this file.

Reply:
Thanks! Done